datastore/store.go

```diff
@@ -186,17 +186,3 @@ func (bs *BlobStore) Get(hint Hint, key []byte) ([]byte, error) {
 	}
 	return nil, fmt.Errorf("blob store not found %s", hint)
 }
-
-func getHeader(atx *types.VerifiedActivationTx) *types.ActivationTxHeader {
```

Comment:
why remove this?

the motivation to keep private here is that header should be cached here. being public in common pkg encourage new code to freely create instances of the header.

Reply:
i am using ActivationHeader in OnAtx. so i added Header() method, can change that if there is something better

Reply:
i am using it here https://github.com/spacemeshos/go-spacemesh/blob/68066ef3cc6c7676b5f9b3e5b76bcfe8818f9fe8/activation/handler.go#L449

not sure why it makes sense to call cdb if data is available right there

Reply:
it's a bit awkward yes. 
if you can find a way not having to create Header instances in common/activation pkg, i'm fine with whatever.
i only care that future dev don't easily use a API like that and effectively remove the use of the header cache.

Reply:
i will use cdb.GetAtxHeader then